SUILesson7_9/SUILesson7_9/MainScreenView.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ContentView.swift
+//  SUILesson7_9
+//
+//  Created by Григоренко Александр Игоревич on 23.01.2023.
+//
+
+import SwiftUI
+
+/// Представление навигации.
```

Comment:
```suggestion
/// Представление навигации.
```
непонятно какой именно навигации